draft-ietf-gnap-core-protocol.md

```diff
@@ -653,24 +702,82 @@ expired access token at the AS using the token's management URL.
     [access token](#response-token) usable at the RS. The access token
     response includes a token management URI.
     
-3. The client instance [presents the token](#use-access-token) to the RS. The RS 
-    validates the token and returns an appropriate response for the
+3. The client instance [presents the token](#use-access-token) to the RS. 
+
+4. The RS validates the token and returns an appropriate response for the
     API.
-    
-4. When the access token is expired, the RS responds to the client instance with
-    an error.
-    
-5. The client instance calls the token management URI returned in (2) to
+ 
+5. Time passes and the client instance presents the token to the RS again.
+   
+6. The RS validates the token and determines that the access token is expired
+    The RS responds to the client instance with an error.
+
+7. The client instance calls the token management URI returned in (2) to
     [rotate the access token](#rotate-access-token). The client instance
     presents the access token as well as the appropriate key.
 
-6. The AS validates the rotation request including the signature
+8. The AS validates the rotation request including the signature
     and keys presented in (5) and returns a 
     [new access token](#response-token-single). The response includes
     a new access token and can also include updated token management 
     information, which the client instance will store in place of the values 
     returned in (2).
-   
+
+### Requesting User Information {#sequence-user}
```

Comment:
It may make sense to mention that the AS authorization server MAY limit the amount of information returned in this flow.

Reply:
These are examples and so adding a normative requirement isn't appropriate, but I can add a note in the process list. Any normative requirements should be in the section on subject information (response-subject).